Memeolist/Memeolist/AppDelegate.swift

```diff
@@ -15,17 +15,23 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
 
         // create the authentication config
         let authenticationConfig = AuthenticationConfig(redirectURL: "memeolist://callback")
-        try! AgsAuth.instance.configure(authConfig: authenticationConfig)
-        if let transport = AgsSync.instance.transport {
-            transport.headerProvider = AgsAuth.instance.getAuthHeaderProvider()
-        }
-        guard let _ = try! AgsAuth.instance.currentUser() else {
-            let rootViewController = LoginViewController()
-            window?.rootViewController = UINavigationController(rootViewController: rootViewController)
-            window?.makeKeyAndVisible()
-            return true
+        do {
+            try AgsAuth.instance.configure(authConfig: authenticationConfig)
+            if let transport = AgsSync.instance.transport {
+                transport.headerProvider = AgsAuth.instance.getAuthHeaderProvider()
+            }
+            guard let _ = try! AgsAuth.instance.currentUser() else {
```

Comment:
Let remove `!` from here as well as this is going to crash app
See:https://github.com/aerogear/aerogear-ios-example-apps/pull/12#issuecomment-420933679

Reply:
yes, I already did.